src/Exploded.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ElegantBro\Arrayee;
+
+use ElegantBro\Interfaces\Arrayee;
+use ElegantBro\Interfaces\Stringify;
+use Exception;
+
+class Exploded implements Arrayee
+{
+    /**
+     * @var Stringify glue
+     */
+    private $glue;
+
+    /**
+     * @var Stringify stringify to explode
+     */
+    private $stringify;
+
+    public static function byComma(Stringify $stringify): Exploded
+    {
+        return new static(new \ElegantBro\Stringify\Just(","), $stringify);
+    }
+
+    public function __construct(Stringify $glue, Stringify $stringify)
```

Comment:
`glue` is for implode operation, for explode it should be `separator`

Reply:
Done!